status-im/status-react
# fix can't pair with dApp on android with WalletConnect

fixes https://github.com/status-im/status-mobile/issues/20083


### Summary

It seems this problem of android not pairing successfully was in `2.12.2` version and migrating to `2.11.2` fixes the issue.
(Thank you @shivekkhurana for [suggestion](https://discord.com/channels/1210237582470807632/1217175655729467492/1256170218095382618) about checking rainbow-wallet, they are also using same version)

But after migrating to, `2.11.2` it started to throw this error - https://github.com/WalletConnect/walletconnect-monorepo/issues/3235
Which was fixed by adding `big-integer` package and assigning that to `BigInt` variable.

<details>

<summary>Video</summary>


https://github.com/status-im/status-mobile/assets/17097240/09c5c86f-b8e4-447f-a466-7cb513c6bda2


</details>

### Testing Step (Only for devs)

1. Connect on https://react-app.walletconnect.com (whatever network)
2. Copy the url
3. Login in the status app
4. Make sure WalletConnect was initialized
5. Run the `:wallet-connect/pair` with the copied url


status: ready
